module_intros/pmft.PMFTXYZ.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<p>Should we maybe just put the warning filter into the PMFT? I feel like we don't want to be throwing warnings when we know why this happens.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>